nextflow_schema.json

```diff
@@ -245,25 +249,15 @@
                     "fa_icon": "far fa-file-code",
                     "description": "Run stringtie analysis"
                 },
-                "whitelist": {
-                    "type": "string",
-                    "fa_icon": "far fa-file-code",
-                    "description": "Path to fusions to add to the input of fusioninspector"
-                },
-                "cosmic_username": {
-                    "type": "string",
+                "tools_cutoff": {
+                    "type": "integer",
```

Comment:
Do we need a validator here? e.g. what happens when --tools_cutoff is 0 ?

Reply:
Resolved in [491501c](https://github.com/nf-core/rnafusion/pull/389/commits/491501c9f6ac97f518ab4a46034581869e8546aa)